blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -68,6 +69,10 @@ public String getId() {
 
     @Override
     public String getDisplayName() {
+        String desc = ArgumentsAction.getArgumentDescriptionString(node.getNode());
```

Comment:
Could we instead move thus to a `getDisplayDescription` so that we can separately display the type of the step and the description?

e.g. the UI should show "Shell: mvn clean install"